packages/fxa-admin-panel/src/components/AccountSearch/index.tsx

```diff
@@ -49,20 +49,82 @@ export const GET_ACCOUNT_BY_EMAIL = gql`
   }
 `;
 
+export const GET_EMAILS = gql`
+  query fetchAll($search: String!) {
+    getEmailsLike(search: $search) {
+      email
+    }
+  }
+`;
+
 export const AccountSearch = () => {
   const [inputValue, setInputValue] = useState<string>('');
   const [showResult, setShowResult] = useState<Boolean>(false);
+  const [suggestions, setSuggestion] = useState<Array<string>>([]);
+  const [text, setText] = useState<string>('');
   const [getAccount, { loading, error, data, refetch }] = useLazyQuery(
     GET_ACCOUNT_BY_EMAIL
   );
+  const [getEmailLike, { data: data1, refetch: refetch1 }] = useLazyQuery(
+    GET_EMAILS
+  );
 
   const handleSubmit = (event: React.FormEvent) => {
     event.preventDefault();
-    getAccount({ variables: { email: inputValue } });
+    getAccount({ variables: { email: text } });
     // Don't hide after the first result is shown
     setShowResult(true);
   };
 
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setInputValue(event.target.value);
+    setSuggestion([]);
+    onTextChanged(event);
+    getEmailLike({ variables: { search: event.target.value } });
```

Comment:
I see that `getEmailLike` is called in `onTextChanged`. Do we still need this one. We shouldn't call it more than necessary.